proto/tendermint/services/block_results/v1/block_results.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package tendermint.services.block_results.v1;
+
+import "tendermint/abci/types.proto";
+import "tendermint/types/params.proto";
+import "gogoproto/gogo.proto";
```

Comment:
We should not use gogoproto, got this feedback from Thane

Reply:
Yup. We should avoid it at all costs to better facilitate a future transition away from it, as per #731.